sample_tracking/app/input-top-bar.tsx

```diff
@@ -0,0 +1,20 @@
+
+import Link from 'next/link';
+
+type InputTopBarProps = {
+    title: string
+}
+
+export default function InputTopBar(props: InputTopBarProps) {
```

Comment:
What is Input Top Bar?  Is that a new UI feature?

Can you explain in comments how this is used?